gidgethub/app.py

```diff
@@ -0,0 +1,44 @@
+"""Support for GitHub Actions."""
+from typing import Any, Dict
+
+import time
+import jwt
+
+from gidgethub.abc import GitHubAPI
+
+
+def get_jwt(*, app_id: str, private_key: str) -> str:
```

Comment:
I realized that there are other endpoints that require JWT, so I changed this from private to public. I've also adjusted the documentation.